PocketKit/Sources/PocketKit/MyList/SavedItemsList/SavedItemsListViewModel.swift

```diff
@@ -371,6 +379,20 @@ class SavedItemsListViewModel: NSObject, ItemsListViewModel {
         tracker.track(event: event, contexts)
     }
 
+    private func trackButton(item: SavedItem, identifier: UIContext.Identifier) {
```

Comment:
decided to create a separate track function and not modify the existing one, let me know if the preference is to modify the existing one and avoid calling `UIContext.myList.item(index: UIIndex(indexPath.item))` for certain identifiers

Reply:
Makes sense to me.